bitwarden.sh

```diff
@@ -190,6 +190,56 @@ function compressLogs() {
     rm $tempfile
 }
 
+function checkSmtp() {
+    CONFIG_FILE="$1/env/global.override.env"
+
+    if [ ! -f "$CONFIG_FILE" ]; then
+        echo "Configuration file not found at $CONFIG_FILE"
+        exit 1
+    fi
+
+    if ! command -v openssl &> /dev/null; then
+        echo "OpenSSL is not available but is required for this check."
+        exit 1
+    fi
+
+    host=$(grep 'globalSettings__mail__smtp__host=' "$CONFIG_FILE" | cut -d '=' -f2)
+    port=$(grep 'globalSettings__mail__smtp__port=' "$CONFIG_FILE" | cut -d '=' -f2)
+    ssl=$(grep 'globalSettings__mail__smtp__ssl=' "$CONFIG_FILE" | cut -d '=' -f2)
+    username=$(grep 'globalSettings__mail__smtp__username=' "$CONFIG_FILE" | cut -d '=' -f2)
+    password=$(grep 'globalSettings__mail__smtp__password=' "$CONFIG_FILE" | cut -d '=' -f2)
+
+    if [ "$ssl" == "true" ]; then
+        ssl_command="-ssl"
+    else
+        ssl_command="-starttls smtp"
+    fi
+
```

Comment:
```suggestion

set +e
```
If the user has not changed the SMTP settings from the defaults on their installation, the `openssl` command will fail and since our script has `set -e` at the top, it will exit immediately without any output. This changes it so that it will at least print the error `SMTP authentication failed or connection error occurred.`